homeassistant/config_entries.py

```diff
@@ -872,6 +875,9 @@ async def _async_init_reconfigure(
             if any(self.async_get_active_flows(hass, {SOURCE_RECONFIGURE})):
                 # Reconfigure flow already in progress for this entry
                 return
+            if any(self.async_get_active_flows(hass, {SOURCE_REAUTH})):
+                # Reauth flow already in progress for this entry
+                return
```

Comment:
I think you can combine these

Reply:
Made the split to make the comment to kick in the "reauth" flow but if we should just return here then yes we combine them.